source/apps/metadata.txt

```diff
@@ -188,7 +192,7 @@ or Atlas CLI to find it programmatically.
                :alt: Expanded ellipses icon showing the Actions menu and the copy
                :width: 400px
 
-   .. tab:: Atlas CLI
+   .. tab:: CLI
```

Comment:
Hmm. I noticed the tab title on the page for the procedure we're un-hiding above is "App Services CLI" and the tab title of this procedure is just "CLI." It looks like we're using `.. tabs-realm-admin-interfaces::` above, and for this procedure, we're just using `.. tabs::` in ln 155. Should we use the same tab group as above for consistency?

Reply:
This procedure can't use the realm-admin-interfaces tabset because it's using all the Atlas stuff. It'd be cool if we could link the App Services CLI and Atlas CLI tabs together but alas they are not currently connected.